glossary.md

```diff
@@ -101,16 +168,32 @@ Currently, available steps generated by PD include:
 - `PromoteLearner`: Promotes a specified learner to a voting member
 - `SplitRegion`: Splits a specified Region into two
 
+### Optimistic transaction
+
+Optimistic transactions are transactions that use optimistic concurrency control and generally do not cause conflicts in concurrent environments. After enabling optimistic transactions, TiDB only checks for conflicts when the transaction is finally committed. The optimistic transaction mode is suitable for concurrent scenarios with more reads and fewer writes, which can improve the performance of TiDB.
+
+Starting from v3.0.8, TiDB clusters default to pessimistic transaction mode. However, if a cluster created before v3.0.7 is upgraded to v3.0.8 or later, the default transaction mode will not be changed; only newly created clusters will default to pessimistic transaction mode. For details, refer to [TiDB Optimistic Transaction Model](/optimistic-transaction.md).
+
 ## P
 
 ### Partitioning
 
-[Partitioning](/partitioned-table.md) refers to physically dividing a table into smaller table partitions, which can be done by partition methods such as RANGE, LIST, HASH, and KEY partitioning.
+[Partitioning](/partitioned-table.md) refers to physically dividing a table into smaller table partitions, which can be done by partition methods such as RANGE, LIST, HASH, and KEY partitioning. These smaller partitions are called partitioned table.
+
+### PD Control (pd-ctl)
+
+PD Control (pd-ctl) is a command-line tool for PD, used to obtain cluster status information and modify the cluster. For details, see [PD Control User Guide](/pd-control.md).
```

Comment:
PD Control should in almost all cases be used via TiUP as that provides updating, downloading, etc. Not sure this entry reflects that well enough.

```
$ tiup ctl:v7.5.0 pd tso 0
Starting component `ctl`: /home/dvaneeden/.tiup/components/ctl/v7.5.0/ctl pd tso 0
system:  1970-01-01 01:00:00 +0100 CET
logic:   0
```

```suggestion
PD Control (pd-ctl) is a command-line tool to interface with the placement driver (PD) of the cluster. This can be used to obtain cluster status information, configuration and modify the cluster. For details, see [PD Control User Guide](/pd-control.md).
```